frontend/src/metabase/actions/containers/ActionCreator/InlineDataReference.styled.tsx

```diff
@@ -20,10 +17,3 @@ export const DataReferenceContainer = styled.div`
     background-color: ${color("white")};
   }
 `;
-
-export const TriggerButton = styled(Button)`
```

Comment:
This isn't needed anymore. Since we're rendering the trigger button inside `<ActionCreatorHeader />`

https://github.com/metabase/metabase/pull/27809/files#diff-a9d477e7189147f253f9e08ed333457267687b625a7eb4fb4db491bcf9ad23e5R19